src/holodeck/spaces.py

```diff
@@ -49,9 +61,18 @@ class ContinuousActionSpace(ActionSpace):
 
```

Comment:
Add low and high to docstring with type list

Reply:
eg ```low (:obj:`list` of :obj:`float` or :obj:`float`): the low```